### ISSUE TITLE
Support CSS-in-JS with postcss-styled-syntax

### ISSUE THREAD

**Issue by EvgenyOrekhov** [AndyOGo/stylelint-declaration-strict-value#326]
I use [postcss-styled-syntax](https://github.com/hudochenkov/postcss-styled-syntax) and I get false warnings.

Config:

```json
{
  "customSyntax": "postcss-styled-syntax",

  "plugins": [
    "stylelint-scales"
  ],

  "rules": {
    "scale-unlimited/declaration-strict-value": [
      "/color$/",
      {
        "ignoreVariables": true,
        "ignoreFunctions": false,
        "ignoreKeywords": ["currentColor", "transparent", "inherit"],
        "disableFix": true
      }
    ]
  }
}
```

Code:

```tsx
import styled from "styled-components";

export const Button = styled.button`
  color: ${(props) => props.theme.colors.button};
`;
```

```text
stylelint --fix '**/*.tsx'

test.tsx
 4:3  ✖  Expected variable, function or keyword for "=>" of "color"  scale-unlimited/declaration-strict-value
```
